Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common
 # install Ruby from source
 #  from source: mainly because of GEM native extensions,
 #  this is the most reliable way to use Ruby no Ubuntu if GEM native extensions are required
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev libsqlite3-dev
```

Comment:
If we're reporting versions for libs then I think it should be consistent & not just libsqlite3. I've updated the PR. Let me know what you think.

> If we preinstall the libs I'd say it might worth to preinstall the sqlite3 package as well.

I don't use sqlite3 directly (only via Ruby bindings). I think we should wait for someone to complain, otherwise we're adding a package that no one uses.